.gitignore

```diff
@@ -13,3 +13,5 @@ test/e2e/cypress/screenshots
 test/e2e/.bin
 pkged.go
 coverage.*
+*.swp
+/vendor
```

Comment:
I don't think we generate *.swp. Also, `/vendor` isn't used. Probably makes sense to revert

Reply:
Actually, I do when using vim and when vim crashed this file stay
But ok I remove it